dash/dash.py

```diff
@@ -541,12 +542,14 @@ def _relative_url_path(relative_package_path="", namespace=""):
 
             modified = int(os.stat(module_path).st_mtime)
 
-            return "{}_dash-component-suites/{}/{}?v={}&m={}".format(
+            return "{}_dash-component-suites/{}/{}".format(
                 self.config.requests_pathname_prefix,
                 namespace,
-                relative_package_path,
-                importlib.import_module(namespace).__version__,
-                modified,
+                build_fingerprint(
+                    relative_package_path,
+                    importlib.import_module(namespace).__version__,
+                    modified,
+                ),
```

Comment:
Isolated the fingerprinting logic with its inverse in a easily testable file / function